<launch>  
    <include file="$(find aruco_detect)/launch/aruco_detect.launch">
        <param name="publish_images" value="true" />
        <arg name="do_pose_estimation" value="true" />
        <arg name="vis_msgs" value="true" />
        <arg name="fiducial_len" value="0.024" />
        <arg name="fiducial_len_override" value="0: 0.035" />
        <arg name="dictionary" value="0" />
        <arg name="camera" value="usb_cam" />
        <arg name="image" value="image_raw" />
    </include>

    <include file="$(find usb_cam)/launch/usb_cam-test.launch" />
    
    <node name="task_list_parser" pkg="assembly_pipeline_framework" type="task_list_parser.py" output="screen"/>

    <node name="asm_line_controller" pkg="assembly_pipeline_framework" type="asm_line_controller.py" output="screen"/>

    <node name="fixed_marker_frame_broadcaster" pkg="assembly_pipeline_framework" type="fixed_marker_frame_broadcaster.py" output="screen"/>

    <node name="camera_frame_broadcaster" pkg="assembly_pipeline_framework" type="camera_frame_broadcaster.py" output="screen" />
    <node name="cargo_frame_broadcaster" pkg="assembly_pipeline_framework" type="cargo_frame_broadcaster.py" output="screen" />
    <node name="workstation_frame_broadcaster" pkg="assembly_pipeline_framework" type="workstation_frame_broadcaster.py" output="screen" />

</launch>
